### ISSUE TITLE
Speed Index Score Confusion

### ISSUE THREAD

**Issue by scottylong** [GoogleChrome/lighthouse#12693]
Hello,

I am using lighthouse for my company's website. My Speed Index score is under 3.0 s, but it appears in red color as it would if it was slow. Could that be contributed to other factors that might be associated with the speed index score? Or is it an error I am getting? 

Regards,
Scott

**Comment by connorjclark** [comment]
hmm, less than 3s should be green, according to our docs https://web.dev/speed-index/

can you share an image of the report or a URL?

**Comment by scottylong** [comment]
> hmm, less than 3s should be green, according to our docs https://web.dev/speed-index/
> 
> can you share an image of the report or a URL?

![image](https://user-images.githubusercontent.com/86424999/123424141-7205ff00-d58e-11eb-8fc1-f86864ba3bc9.png)

**Comment by patrickhulce** [comment]
@scottylong this is a report for desktop, not mobile, isn't it? If so, this is expected behavior. Performance on desktop without any throttling or emulation is expected to be *significantly* faster than the thresholds for a slow mobile page load.

> Could that be contributed to other factors that might be associated with the speed index score?

I'm not sure what this means, but this page looks fairly slow for a desktop load. There is definitely room for improvement and Lighthouse is WAI.

**Comment by scottylong** [comment]
@patrickhulce Yes, this report is for desktop.

![image](https://user-images.githubusercontent.com/86424999/123447358-c1a4f480-d5a7-11eb-9219-144a3f9f43d0.png)

Its expected behavior? 

https://web.dev/speed-index/ 

According to the docs, it should be green regardless.

**Comment by scottylong** [comment]
Ran it again

![image](https://user-images.githubusercontent.com/86424999/123449517-c4a0e480-d5a9-11eb-9bc4-aaf248ec9205.png)

Speed Index is 2.6s now and its still red.

**Comment by patrickhulce** [comment]
> According to the docs, it should be green regardless.

The docs are for the default mobile usage of Lighthouse, and have not been updated to express the difference between desktop/mobile score curves.

**Comment by patrickhulce** [comment]
> Speed Index is 2.6s now and its still red.

Yes, again, this is expected behavior. Anything more than 2.3s will be red and you need less than 1.3s to be green.

https://github.com/GoogleChrome/lighthouse/blob/a5be72c3147cdd456055c7f3ac5a2de65ff22eb4/lighthouse-core/audits/metrics/speed-index.js#L49-L55

**Comment by scottylong** [comment]
Thanks for the clarification! This makes sense now.

**Comment by brendankenny** [comment]
https://github.com/GoogleChrome/web.dev/issues/6904 also mentioned the confusion this caused and the need for clear documentation on the mobile vs desktop curves

**Comment by connorjclark** [comment]
seems resolved now (https://developer.chrome.com/docs/lighthouse/performance/first-contentful-paint etc)
